refactor: add protocol support to `stats/base/variancewd` (#5831)
stdlib-js/stdlib: lib/node_modules/@stdlib/stats/base/variancewd/README.md

-   **N**: number of indexed elements.
-   **correction**: degrees of freedom adjustment. Setting this parameter to a value other than `0` has the effect of adjusting the divisor during the calculation of the [variance][variance] according to `N-c` where `c` corresponds to the provided degrees of freedom adjustment. When computing the [variance][variance] of a population, setting this parameter to `0` is the standard choice (i.e., the provided array contains data constituting an entire population). When computing the unbiased sample [variance][variance], setting this parameter to `1` is the standard choice (i.e., the provided array contains data sampled from a larger population; this is commonly referred to as Bessel's correction).
-   **x**: input [`Array`][mdn-array] or [`typed array`][mdn-typed-array].
-   **strideX**: stride length for `x`.

> The `N` and strideX parameters determine which elements in the strided array are accessed at runtime. For example, to compute the minimum value of every other element in `x`,

Reviewer: ```suggestion
The `N` and stride parameters determine which elements in the strided array are accessed at runtime. For example, to compute the [variance][variance] of every other element in `x`,
```

Beware of copy-paste mistakes.

-   **offset**: starting index for `x`.

> While [`typed array`][mdn-typed-array] views mandate a view offset based on the underlying buffer, the offset parameter supports indexing semantics based on a starting index. For example, to calculate the [variance][variance] for every other element in the strided array  starting from the second element

Reviewer: ```suggestion
While [`typed array`][mdn-typed-array] views mandate a view offset based on the underlying buffer, the offset parameter supports indexing semantics based on a starting index. For example, to calculate the [variance][variance] for every other element in the strided array starting from the second element
```

## Notes

-   If `N <= 0`, both functions return `NaN`.

> -   Both functions support array-like objects having getter and setter accessors for array element access (e.g., [`@stdlib/array/base/      accessor`][@stdlib/array/base/accessor]).

Reviewer: ```suggestion
-   Both functions support array-like objects having getter and setter accessors for array element access (e.g., [`@stdlib/array/base/accessor`][@stdlib/array/base/accessor]).
```

> -   Both functions support array-like objects having getter and setter accessors for array element access (e.g., [`@stdlib/array/base/      accessor`][@stdlib/array/base/accessor]).
> -   If `N - c` is less than or equal to `0` (where `c` corresponds to the provided degrees of freedom adjustment), both functions return `NaN`.

Reviewer: ```suggestion
-   If `N - c` is less than or equal to `0` (where `c` corresponds to the provided degrees of freedom adjustment), both functions return `NaN`.
-   Both functions support array-like objects having getter and setter accessors for array element access (e.g., [`@stdlib/array/base/accessor`][@stdlib/array/base/accessor]).
```

[@stdlib/stats/base/dvariancewd]: https://github.com/stdlib-js/stdlib/tree/develop/lib/node_modules/%40stdlib/stats/base/dvariancewd

> [@stdlib/array/base/accessor]: https://github.com/stdlib-js/stdlib/tree/develop/lib/node_modules/%40stdlib/array/base/accessor

Reviewer: This link should be moved outside of the "related links" section. Move to L251.
